package.json

```diff
@@ -2,19 +2,21 @@
   "name": "cosmiconfig",
   "version": "2.2.2",
   "description": "Find and load configuration from a package.json property, rc file, or CommonJS module",
-  "main": "index.js",
+  "main": "dist/index.js",
   "files": [
     "index.js",
-    "lib"
+    "dist"
   ],
   "scripts": {
     "precommit": "lint-staged && jest",
```

Comment:
We should add the flow types check in pre-commit hook as well.

Reply:
Sure.